pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/copy/CopyExecutor.java

```diff
@@ -56,13 +60,32 @@ public void execute(Handle handle, long operatorId, long participantId, CopyConf
             CopyLocation target = pair.getTarget();
 
             if (source.getType() == CopyLocationType.ANSWER && target.getType() == CopyLocationType.ANSWER) {
+                QuestionDto sourceQuestion;
+                FormResponse sourceInstance;
                 String sourceStableId = ((CopyAnswerLocation) source).getQuestionStableId();
-                QuestionDto sourceQuestion = questionDtosByStableId.get(sourceStableId);
-                if (sourceQuestion == null) {
-                    continue; // Question might have been removed from activity, so we skip it.
+                String user = ((CopyAnswerLocation) source).getUser();
+                if (UserType.OPERATOR.equalsIgnoreCase(user)) {
```

Comment:
@LKapanad :
Trying to understand scenario where proxy/operator has question answers that need to be copied to aged-up user ? Does lms consent have question answers by operator that need to be copied to aged-up user ?
In existing studies we are creating new consent for aged-up users, trying to understand what is it different for lms !

Reply:
@ssettipalli the difference is that the parent fills the child's country and state in PREQUAL activity and this data gets lost for child after ageup. Yes consent is generated for ageup participant too but there's no question about country. which is needed in the post-consent activity consent_addendum